client/public/index.html

```diff
@@ -5,6 +5,7 @@
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
+    <script src="https://kit.fontawesome.com/aee911f74a.js" crossorigin="anonymous"></script>
```

Comment:
Material UI has an icons library that's fairly complete, I used a few of those icons in the prospects upload and gmail dialog. I think we should probably stick to one icons library for consistency, were there any icons you needed that are missing from the MUI icons?

Reply:
Right!